rts/Game/Camera.h

```diff
@@ -244,6 +249,28 @@ class CCamera {
 
 	int viewport[4];
 
+	/**
+	* @brief moveFastMult
+	* The multiplier applied to speed when camera is in movefast state.
+	*/
+	float moveFastMult;
+
+	/**
+	* @brief moveSlowMult
+	* The multiplier applied to speed when camera is in moveslow state.
+	*/
+	float moveSlowMult;
+
+	/**
+	* @brief moveFastScale
+	* Scaling for CameraMoveFastMult.
+	*/
+	float moveFastScale;
+
+	int edgeMoveWidth;
```

Comment:
Forgot to add to listener.

Reply:
I remember thinking that since they weren't listened to before, they shouldn't be now, but they were re-queried every time which amounts to same result. Will add both of these and the ones in the other comment.